services/ui-src/src/forms/mcpar/dpi/dpi.json

```
@@ -1,18 +1,181 @@
+                    "mask": "comma-separated"
+                  }
+                }
+              ]
+            },
+            {
+              "name": "dpi-6-o2",
+              "type": "choice",
+              "label": "Makes referrals to the State Medicaid Agency (SMA) and MFCU concurrently",
+              "children": [
+                {
+                  "id": "dpi-7-o2",
+                  "type": "number",
+                  "props": {
+                    "label": "D1.X.7 Count of program integrity referrals to the state",
+                    "hint": "Enter the count of program integrity referrals that the plan made to the state in the past year. Enter the count of unduplicated referrals",
+                    "mask": "comma-separated"
+                  }
+                }
+              ]
+            },
+            {
+              "name": "dpi-6-o3",
+              "type": "choice",
+              "label": "Makes some referrals to the SMA and others directly to the MFCU",
+              "children": [
+                {
+                  "id": "dpi-7-o3",
+                  "type": "number",
+                  "props": {
+                    "label": "D1.X.7 Count of program integrity referrals to the state",
+                    "hint": "Enter the count of program integrity referrals that the plan made to the state in the past year. Enter the count of referrals made to the SMA and the MFCU in aggregate.",
+                    "mask": "comma-separated"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        "id": "dpi-8",
+        "type": "number",
+        "props": {
+          "label": "D1.X.8 Ratio of program integrity referrals to the state",
+          "hint": "What is the ratio of program integrity referrals listed in the previous indicator made to the state in the past year per 1,000 beneficiaries, using the plan's total enrollment as of the first day of the last month of the reporting year (reported in indicator D1.I.2) as the denominator.",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-9",
+        "type": "textarea",
+        "props": {
+          "label": "D1.X.9 Plan overpayment reporting to the state",
+          "hint": "Describe the plan’s latest annual overpayment recovery report submitted to the state as required under 42 CFR 438.608(d)(3).</p>Include, for example, the following information:<ul><li>The date of the report (rating period or calendar year).</li><li>The dollar amount of overpayments recovered.</li><li>The ratio of the dollar amount of overpayments recovered as a percent of premium revenue as defined in MLR reporting under 438.8(f)(2).</li></ul>"
+        }
+      },
+      {
+        "id": "dpi-10",
+        "type": "radio",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "",
+          "hint": "",
```

Oops. Just forgot to fill these in.

```suggestion
          "label": "D1.X.10 Changes in beneficiary circumstances",
          "hint": "Select the frequency the plan reports changes in beneficiary circumstances to the state.",
```